.github/wiki/Getting-Started.md

```diff
@@ -131,20 +131,27 @@ $global:lastRepository = $null
 function Check-DirectoryForNewRepository {
     $currentRepository = git rev-parse --show-toplevel 2>$null
     if ($currentRepository -and ($currentRepository -ne $global:lastRepository)) {
-        onefetch
+        if ([Console]::OutputEncoding -ne [System.Text.Encoding]::UTF8) {
+            [Console]::OutputEncoding = [System.Text.Encoding]::UTF8
+        }
```

Comment:
Is it perhaps too opinionated to set the output encoding? Is there an issue that this resolves?

Reply:
Yep, so basically as I mentioned if don't pipe to `Write-Host` nothing prints via onefetch if i `z` into a git repo, but if I don't send the console encoding some symbols get garbled, for example the circles, also just added that if statement to avoid setting the encoding multiple times redundantly

![image](https://github.com/user-attachments/assets/7d9eb180-2380-473f-80d8-18dd196c7232)

Reply:
Oh, makes sense. Perhaps we could recommend users set their encoding in their `$PROFILE`,  instead of in this function? Seems like something that belongs in startup configuration. What do you think?

Reply:
How about we add a comment regarding this, or just it globally above this line?

```pwsh
$global:lastRepository = $null
```

Reply:
Yeah, setting it globally makes sense, and maybe a comment explaining why they need to do that.